build/export-pipeline.yml

```diff
@@ -37,69 +36,163 @@ stages:
         arguments: '--configuration $(buildConfiguration) /warnaserror -f $(defaultBuildFramework)'
     - template: ./jobs/package-integration-tests.yml
 
-- stage: redeployStu3
-  displayName: 'Redeploy STU3 Site'
+- stage: exportRedeployStu3
+  displayName: 'Redeploy Export STU3 Site'
   dependsOn: []
+  variables:
+  - template: export-variables.yml
   jobs:
   - template: ./jobs/restart-webapp.yml
     parameters: 
       webAppName: $(DeploymentEnvironmentName)
       subscription: $(ConnectedServiceName)
 
-- stage: redeployStu3Sql
-  displayName: 'Redeploy STU3 SQL Site'
+- stage: exportRedeployStu3Sql
+  displayName: 'Redeploy Export STU3 SQL Site'
   dependsOn: []
+  variables:
+  - template: export-variables.yml
   jobs:
   - template: ./jobs/restart-webapp.yml
     parameters: 
       webAppName: $(DeploymentEnvironmentNameSql)
       subscription: $(ConnectedServiceName)
 
-- stage: redeployR4
-  displayName: 'Redeploy R4 Site'
+- stage: exportRedeployR4
+  displayName: 'Redeploy Export R4 Site'
   dependsOn: []
+  variables:
+  - template: export-variables.yml
   jobs:
   - template: ./jobs/restart-webapp.yml
     parameters: 
       webAppName: $(DeploymentEnvironmentNameR4)
       subscription: $(ConnectedServiceName)
 
-- stage: redeployR4Sql
-  displayName: 'Redeploy R4 SQL Site'
+- stage: exportRedeployR4Sql
+  displayName: 'Redeploy Export R4 SQL Site'
   dependsOn: []
+  variables:
+  - template: export-variables.yml
   jobs:
   - template: ./jobs/restart-webapp.yml
     parameters: 
       webAppName: $(DeploymentEnvironmentNameR4Sql)
       subscription: $(ConnectedServiceName)
 
-- stage: testStu3
-  displayName: 'Run Stu3 Tests'
+- stage: importRedeployStu3
```

Comment:
Why would we redeploy the Cosmos services? Import doesn't run on them.

Reply:
I couldn't remember if we did or not. I'll remove them.

Reply:
Removed